src/main/java/com/ericsson/ei/subscription/EmailSender.java

```diff
@@ -58,22 +58,32 @@
     @Autowired
     private JavaMailSender emailSender;
 
+    @PostConstruct
+    public void logSettings() {
+        LOGGER.debug("Email Sender : " + sender);
+        LOGGER.debug("Email Subject : " + subject);
```

Comment:
Merge these two logs to one LOGGER.debug message.

Reply:
They can also be written using brackets and parameters instead of concat: LOGGER.debug("Email Sender : {}", sender);

Reply:
Since we have a separate class ConfigurationLogger already in use, I think we can remove these log methods with @Postconstruct annotation on them, right?

Reply:
Yes. Make sure those settings is logged.